src/sage/geometry/polyhedron/combinatorial_polyhedron/base.pyx

```diff
@@ -3771,117 +3727,36 @@ cdef class CombinatorialPolyhedron(SageObject):
                 dimension_one += 1
             dimension_two = -1
 
-        try:
-            incidences = <size_t**> check_malloc(sizeof(size_t*))
-            while (dimension_one < dim + 1):
-                already_seen = sum(f_vector[j] for j in range(dimension_two + 1))
-                already_seen_next = already_seen + f_vector[dimension_two + 1]
+        while (dimension_one < dim + 1):
```

Comment:
```suggestion
        while dimension_one < dim + 1:
```

Reply:
Done in 2367bc9685579e0a554a1371f6b919e532ab6845.